### ISSUE TITLE
Integrate webpack bin

### ISSUE THREAD

**Issue by jhnns** [webpack/webpack.js.org#264]
Have we talked about integrating [webpack bin](http://www.webpackbin.com/EkWNOqxyG) yet? I think it would be cool to get an interactive playground on the site without needing to install anything. We talked about that a little bit at #32 already.

Maybe @christianalfoni wants to join that discussion?

**Comment by bebraw** [comment]
Related commentary: https://github.com/webpack/webpack.js.org/pull/222#issuecomment-253502068 .

**Comment by bebraw** [comment]
If you check #222, you can see we can get something limited fairly easily up and running. It's not the same, but I have a feeling it would get us close to a cool solution without having to rely on a third party service and while keeping it light.

**Comment by jhnns** [comment]
(Sorry, I forgot that I mentioned it already ^^. I searched for webpack bin in our issues but couldn't find anything)

Can you bundle webpack with webpack 😀 ?

**Comment by skipjack** [comment]
If anyone wants to see a quick example of what webpackbin would look like on the splash page, or maybe in Getting Started would be better, you can grab the iframe that I removed in 2c6c1563609 to get a general idea. I know we don't want to go with an `<iframe>` long term but it just gives a rough idea of what an embed might look like.

If we do decide to go custom, I think we'd just have to flesh out what @pksjce did in #222. I like the idea of a simple, relatively static example on the splash/landing page (like we have) with a more robust, possibly interactive example in **Getting Started**.

**Comment by bebraw** [comment]
What would you think if we skipped this one? There are [tons of open issues](https://github.com/christianalfoni/webpack-bin/issues) and webpackbin would require update to webpack 2 for this to make any sense.

It would be likely smarter for us to build something limited for demonstration purposes like @pksjce did earlier. We can get something interactive going.

In any case this is a secondary issue.

**Comment by pksjce** [comment]
Would definitely work on this if the decision made is favorable!

**Comment by skipjack** [comment]
Closing this for now as we've moved away from this idea a bit. I really like webpackbin and am even starting to use it instead of codepen from time to time, but, as mentioned before, there [hasn't been much activity in that repo recently](https://github.com/christianalfoni/webpack-bin/commits/master). I think we should re-open down the road if anything.

@pksjce let me know if you start working on your interactive editor components again for the splash or getting-started page. I'd be happy to help with design or whatever else.
